package/MDAnalysis/core/topologyobjects.py

```diff
@@ -107,9 +107,14 @@ def type(self):
     def is_guessed(self):
         return bool(self._guessed)
 
+    # TODO: which one to use?
```

Comment:
Yeah sorting indices isn't good actually, `(2, 1, 3)` is a different angle to `(1, 2, 3)`.  They can only get 'mirrored' `(2, 1, 3) == (3, 1, 2)`.  So yeah I was wrong, lets use the first one with Universe in it